worlds/ladx/docs/en_Links Awakening DX.md

```diff
@@ -0,0 +1,79 @@
+# Links Awakening DX
+
+## Where is the settings page?
+
+The [player settings page for this game](../player-settings) contains all the options you need to configure and export a
+config file.
+
+## What does randomization do to this game?
+
+Items which the player would normally acquire throughout the game have been moved around. Logic remains, so the game is
+always able to be completed, but because of the item shuffle the player may need to access certain areas before they
+would in the vanilla game.
+
+## What items and locations get shuffled?
+
+All main inventory items, collectables, and ammunition can be shuffled, and all locations in the game which could
+contain any of those items may have their contents changed.
+
+## Which items can be in another player's world?
+
+Any of the items which can be shuffled may also be placed into another player's world. It is possible to choose to limit
+certain items to your own world.
+
+## What does another world's item look like in Link's Awakening?
+
+The game will try to pick an appropriate sprite for the item (a LttP sword will be a sword!) - it may however be a little odd (a Missile Pack may be a single arrow).
```

Comment:
```suggestion
The game will try to pick an appropriate sprite for the item (a LttP sword will be a sword!) - it may, however, be a little odd (a Missile Pack may be a single arrow).
```